tests/suites/test_suite_psa_crypto_metadata.data

```diff
@@ -293,15 +293,27 @@ key_type:PSA_KEY_TYPE_DERIVE:KEY_TYPE_IS_UNSTRUCTURED
 
 Block cipher key type: AES
 depends_on:PSA_WANT_KEY_TYPE_AES
-block_cipher_key_type:PSA_KEY_TYPE_AES:16
+block_cipher_key_type:PSA_KEY_TYPE_AES:16:0
 
 Block cipher key type: DES
 depends_on:PSA_WANT_KEY_TYPE_DES
-block_cipher_key_type:PSA_KEY_TYPE_DES:8
+block_cipher_key_type:PSA_KEY_TYPE_DES:8:0
 
 Block cipher key type: Camellia
 depends_on:PSA_WANT_KEY_TYPE_CAMELLIA
-block_cipher_key_type:PSA_KEY_TYPE_CAMELLIA:16
+block_cipher_key_type:PSA_KEY_TYPE_CAMELLIA:16:0
+
+Block cipher key type: AES non power 2 block_size  (24)
+depends_on:PSA_WANT_KEY_TYPE_AES
+block_cipher_key_type:PSA_KEY_TYPE_AES:24:1
```

Comment:
The added test cases don't make sense. The block size of AES is 16, and therefore the block size that the library thinks AES has must be 16. Please remove them.

There are additional test scripts that parse `test_suite_psa_crypto_metadata.data` to obtain information about supported algorithms. They don't currently care about block cipher block sizes, but if they did, these fake test cases would be not only superfluous but even harmful.

Reply:
That makes sense but how would you suggest I go about testing my function actually works? Like I know it will successfully detect non powers of 2 but in practice the block sizes used are always powers of 2, so I am not sure how to input any bad data which does not trip up the other check for the correct block size per the data type

Reply:
Test it by temporarily adding bad data, but don't commit that.